packages/twenty-server/src/engine/core-modules/workflow/dtos/workflow-trigger-result.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+import { IsObject } from 'class-validator';
+import graphqlTypeJson from 'graphql-type-json';
+
+@ObjectType('WorkflowTriggerResult')
+export class WorkflowTriggerResultDto {
```

Comment:
let's remain consistent with the rest of the code. We usually do not put Dto in the name, or we put DTO